bloomberg/xcdiff
# Add copy files build phase comparator

Resolves: https://github.com/bloomberg/xcdiff/issues/17

**Describe your changes**
Added copy files build phase comparator which allow to find differences in "Extensions" or "Embed Frameworks" build phases. The comparator only compares common build phases (build phases with the same name in both projects). Name or order differences can be picked up by `BuildPhasesComparator`.

**Testing performed**
- CI / unit and integration tests
- Inspect command tests output
- Copy an existing project, edit one of the copy files build phases in one of the projects, make sure all the changes are reported when running (`xcdiff -g copy_files`):
  - add a file
  - modify file properties
  - update properties i.e. destination path
